runtime/src/lib.rs

```diff
@@ -88,7 +88,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("node-template"),
 	impl_name: create_runtime_str!("node-template"),
 	authoring_version: 1,
-	spec_version: 1,
+	spec_version: 100,
```

Comment:
what is this version bump about?

Reply:
Good question, I thought this was something you and @haerdib coordinated.

Reply:
An adaption to the changes substrate has done on their node-template: 
https://github.com/paritytech/substrate/blob/master/bin/node-template/runtime/src/lib.rs#L100

resp, the PR: 
https://github.com/paritytech/substrate/pull/8473

Reply:
Should we change back to 1?

Reply:
Well in our case this is currently only informative, but I still think 1 is better. 100 seems a bit random.

Reply:
If the comment
> 	// This value is set to 100 to notify Polkadot-JS App (https://polkadot.js.org/apps) to use
>	//   the compatible custom types.
is the only reason to bump it I'm not sure it serves us. What "custom types" are these? Just template stuff?

I'm in favor of `1`